ResonatorNetwork.sc

```diff
@@ -4,7 +4,7 @@ ResonatorNetwork {
 
 	*initClass {
 		Class.initClassTree(String);
-		pythonScriptPath = this.class.filenameSymbol.asString.dirname ++ "/python";
+		pythonScriptPath = this.class.filenameSymbol.asString.dirname;
```

Comment:
The _modalData.json_ and _networkArgs.json_ files are now placed in the root of the project and git will interpret them as untracked files due to this change, so they should probably be added to .gitignore.

The whole "old" _python_ sub directory could be removed from .gitignore actually.

Reply:
About modalData and networkArgs, I thought it could be better to use Platform.defaultTempDir if the files aren't needed or the path is not set to an specific directory, but I didn't make changes in sclang classes if it was no needed for python.

Reply:
I believe this suggested change was added, I can't see ``++ "/python"`` in my local clone.